third_party/xla/third_party/tsl/third_party/hwloc/hwloc.BUILD

```diff
@@ -272,6 +272,10 @@ cc_library(
             "hwloc/topology-linux.c",
             "include/hwloc/linux.h",
         ],
+        "@local_xla//xla/tsl:linux_s390x": [
```

Comment:
I am not 100% sure whether this will work in one PR. It could be that you might need to create a separate PR for the OpenXLA repository: https://github.com/openxla/xla

Reply:
I can confirm, the change in this file is lost when importing from this PR, as it changes XLA. Please create another PR for this.

Reply:
Thank you for reviewing @akuegel , PRs have been auto created in tsl as well as XLA by copybara-service and linked below.

Reply:
That is true, but in order to get the OpenXLA change merged, it needs to be imported into our internal system. And that import only works if the PR was sent to the right repository to begin with. I checked and internally, we have only one change we can approve to get it merged, and that is the one that changes tensorflow.

Reply:
ok, so should I remove this change from this PR? and keep only this `third_party/hwloc/hwloc.BUILD` ?

Reply:
This change can remain like it is. Once it will be merged, it will contain just the changes to Tensorflow. The process is like this: we import from Github, then it is approved internally, then it is exported back to Github. Given that the imported change only has the tensorflow bits, then also those will be exported.

Reply:
Actually, what surprises me that indeed the OpenXLA part got merged as well, although in the internal change it was invisible. Not sure whether it is working as intended (from an internal point of view, as the internal reviewer does not see everything that gets merged), but at least we got it merged : https://github.com/openxla/xla/commit/6772834e77115e7368a418ef71024540274c93b2

Reply:
so it landed as part of https://github.com/openxla/xla/pull/22935 PR? and not via this PR?

Reply:
As far as I can see, it got merged into Tensorflow via this PR, and into OpenXLA with an automatically created PR (the one that you had already seen referenced). I was a bit surprised by the behavior, because I thought we re-export the internal change, but seems this is not what is happening. It is possible that there is some difference in behavior between Tensorflow repository and OpenXLA repository. I remember that we sometimes "lost" changes to TSL when a PR was created that changed both OpenXLA and TSL, so I thought the same would happen here.

Reply:
ok, as you pointed out changes are now present in tsl, openxla as well as TensorFlow.
Thank you for your support!